README.md

```diff
@@ -1,2 +1,30 @@
 # clamav-http
 Clamav instance with http api. To supersede https://github.com/UKHomeOffice/docker-clamav
+
+## Installation
+
+Basic installation can be acheived by running:
+
+```
+helm install -n <namespace> clamav ./charts/clamav
+```
+
+Clamav will be installed in the namespace and available at https://clamav/
+
+More detailed documentation on the helm chart can be found [here](/charts/clamav/README.md)
+
+## Components
+
+clamav-http is made up of three components, clamav, clamav-http, and clamav-notify and is designed to be deployed as a service in kubernetes via its helm chart.
+
+### clamav
+
+An extremely barebones clamav/freshclam image with no config. Expects configuration files to be provided via kubernetes configmaps.
+
+### [clamav-http](/clamav-http/README.md)
+
+Written in golang, provides an http based api to clamav
+
+### clamav-notify
```

Comment:
Would be good to mention the RBAC that this needs to work

Reply:
Adding this now